### ISSUE TITLE
Cannot install the drivers (Version 1.4.116)

### ISSUE THREAD

**Issue by Veganlol** [Ryochan7/DS4Windows#242]
Hai, so I'm trying to install these drivers since a week now, but nothing seem to work here.
So when I open DS4Windows.exe then go to the settings tab and click on that "Controller / Driver Setup" label, I click on install DS4 drivers on the following window, it loads 1 or 2 seconds and says installed. 

But the controller still won't connect. Neither do I have a sound when I plug it into my PC, nor a "Wireless Controller" device is showing up. I also don't have that "HID compliant game controller" showing up in my device manager, if that is even important but I think it is. It's not disabled or something (I've read some issues with that) it's just completely missing.

Also tried disabling driver signature verification.

I'm on Windows 10 with latest drivers and using DS4Windows v.1.4.116. So the latest as I'm writing this.
I think it is not installing drivers correctly, right?

Can anyone help me? Am I missing something?

**Comment by Ryochan7** [comment]
It sounds like you are having issues with getting your controller working with Windows in general. It would be better just to make sure you can sync your controller to your PC and make sure Windows can recognize it before attempting to try DS4Windows again. If Windows cannot detect the controller then DS4Windows is not going to be able to do anything. The controller not working when it is connected via USB is a big problem as that option should almost always work.

As for whether the Scp Virtual Bus Driver package installed in the first place, you can look in the Windows Device Manager for a device at System devices -> Scp Virtual Bus Driver. Also, having no Welcome dialog window appear when launching DS4Windows should also mean that the driver installed successfully.

The driver is actually used for the Xbox 360 controller emulation portion of DS4Windows. It is not something that is needed in order for Windows to recognize a DS4 when one is connected to your PC. Windows can use a DS4 as a DirectInput controller but using DirectInput mode is a bit limiting.

**Comment by Veganlol** [comment]
Thanks for the answer.

I finally got it to work. I've used the Scp Toolkit by Nefarius for my PS3 controller before and it seems something messed up the Xbox 360 drivers on my machine that came with Win10.

So for everyone for having similiar issues, here's what I did. I removed the Toolkit completely by running the Wipe Tool that came with it. Rebootet. Then ran Windows Update and it installed the Xbox drivers again. Once installed, my PS4 controller finally shown up.

I just had no Xbox drivers installed, or it was broken or something. Reinstalling Xbox 360 drivers through Windows Update fixed everything for me.

Working like a charm now! :)
